dist/components/Calendar/Body/Body.d.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
```

Comment:
I would recommend deleting the `dist` folder and committing the deleted files. Then add `dist` back to the `.gitignore`. That would make this PR much easier to review and remove all the generated noise.

Reply:
I did this